daemon/execdriver/native/create.go

```diff
@@ -244,6 +282,18 @@ func (d *driver) setupMounts(container *configs.Config, c *execdriver.Command) e
 			flags |= syscall.MS_SLAVE
 		}
 
+		// for user-namespaces, any bind-mounted content which is owned by root needs
+		// to be remapped to the container's root, if user namespacing is turned on
+		srcStat, err := system.Stat(m.Source)
+		if err != nil {
+			return err
+		}
+		if c.RemappedRoot.Uid != 0 && srcStat.Uid() == 0 {
+			if err := os.Chown(m.Source, c.RemappedRoot.Uid, c.RemappedRoot.Gid); err != nil {
```

Comment:
Ugh..this is incorrect.  I merged in an initial "hack" about volumes, but this piece was not supposed to be permanent.  The only chown of content outside the container that is required is the special mounts in `/var/lib/docker/*/containers/<id>/{resolv.conf,hostname,hosts} so that they remain writeable as expected in current Docker behavior.  I will fix this in an update.

Instead, the current thinking was that it is "an exercise for the user" to handle volume/bind-mounts as far as ownership to align with what the operator already has set up with regard to remapping.